build_darwin.sh

```diff
@@ -1,15 +1,48 @@
 #!/bin/sh
 set -eu
 
-curl -fsSL "https://www.torproject.org/dist/tor-0.3.2.10.tar.gz" -o tor-0.3.2.10.tar.gz
-curl -fsSL "https://www.torproject.org/dist/tor-0.3.2.10.tar.gz.asc" -o tor-0.3.2.10.tar.gz.asc
+curl -fsSL "https://www.openssl.org/source/openssl-$OPENSSL_VERSION.tar.gz" -o openssl-$OPENSSL_VERSION.tar.gz && \
+echo "$OPENSSL_HASH  openssl-$OPENSSL_VERSION.tar.gz" | shasum -a 256 -c - && \
+tar -xvzf openssl-$OPENSSL_VERSION.tar.gz && \
+cd openssl-$OPENSSL_VERSION && \
+./config --prefix=$PWD/install no-shared no-dso && \
+make
```

Comment:
`make test` for darwin fails because of `expired certs`. Some forums say that it is because of a conflict with xcode and brew.

Reply:
OK.  Can you put a comment in here explaining this, and quoting the error so we can see if it's still around in the future?

Reply:
https://github.com/brave/tor_build_scripts/issues/6